test/test_symbol.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from src.symbol import SymbolOptions
+
+
+class TestSymbolOptions(unittest.TestCase):
+    def test_get_symbols_returns_symbol_1_if_1_is_inputted(self):
```

Comment:
this would be a great place for a property test to dynamically test all the symbols.